test/looping.js

```diff
@@ -128,14 +128,14 @@ describe('Looping', function() {
   });
 
   it('should ignore loop with invalid through attribute (looping/undefinedObjectLoop.html)', function(done) {
+    teddy.setVerbosity(2);
     assert.equalIgnoreSpaces(teddy.render('looping/undefinedObjectLoop.html', model), '<div></div>');
+    teddy.setVerbosity(1);
```

Comment:
can we make mocha suppress the display of the console output while still triggering the warn code path?